supernode/services/artworkregister/task.go

```diff
@@ -378,6 +638,12 @@ func (task *Task) ValidatePreBurnTransaction(ctx context.Context, txid string) (
 						return nil
 					}
 
+					if err = task.storeIDFiles(ctx); err != nil {
```

Comment:
I don't think storeFingerprints() in line 635 is necessary any more?

Reply:
i'm not sure

Reply:
ok i've removed it after confirmation